sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -40,7 +40,9 @@ import org.apache.spark.sql.test.SharedSparkSession
 import org.apache.spark.sql.types.{IntegerType, StructField, StructType}
 
 class DDLParserSuite extends AnalysisTest with SharedSparkSession {
-  private lazy val parser = new SparkSqlParser(new SQLConf)
+  private lazy val parserConf = new SQLConf
+  parserConf.setConf(SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED, false)
```

Comment:
This test suite heavily depends on `LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED=false`.

Reply:
nit:
```
private lazy val parser = new SparkSqlParser(new SQLConf().copy(
  SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED -> false))
```

Reply:
Thanks!